src/interfaces/packagingSObjects.ts

```diff
@@ -267,7 +267,7 @@ export namespace PackagingSObjects {
     EnableRss: boolean;
     UpgradeType: Nullable<'delete-only' | 'deprecate-only' | 'mixed-mode'>;
     ApexCompileType: Nullable<'all' | 'package'>;
-    SkipHandlers: boolean;
+    SkipHandlers: string;
```

Comment:
@KSCCO The contents of packagingSObjects.ts is meant to reflect the actual shape of the server side objects. You had mentioned that this property is actually a string array on the server side. If this is correct, then this should be a string[].

Reply:
@peternhale, the corresponding server side object is just a string but does get parsed as a comma separated string. 
https://sourcegraph.soma.salesforce.com/perforce.soma.salesforce.com/app/main/core@HEAD/-/blob/udd-xml/java/resources/udd/PackageInstallRequest.entity.xml?L42

Reply:
Thanks for clarifying @KSCCO.